src/components/HelpMenu/index.tsx

```diff
@@ -89,26 +87,26 @@ const HelpMenu = (): JSX.Element => {
         {
             key: "version",
             label: (
-                <a
-                    onClick={() => {
+                <NavButton
+                    titleText={"Version info"}
+                    clickHandler={() => {
                         setModalVisible(!modalVisible);
                     }}
-                >
-                    Version info
-                </a>
+                    buttonType={ButtonClass.DropdownItem}
+                />
             ),
         },
     ];
 
     return (
         <>
-            <Dropdown menu={{ items, theme: "dark", className: styles.menu }}>
-                <NavButton
-                    titleText={"Help "}
-                    icon={DownArrow}
-                    buttonType={ButtonClass.Secondary}
-                />
-            </Dropdown>
+            <CustomDropdown
+                items={items}
+                titleText={"Help "}
```

Comment:
Should `CustomDropdown`/`NavButton` just add spacing between the titleText and the icon?

Reply:
Why don't we just remove that silly space instead.